var/spack/repos/builtin/packages/geant4-data/package.py

```diff
@@ -136,8 +136,10 @@ class Geant4Data(BundlePackage):
 
     for _vers, _dsets in _datasets.items():
         _vers = "@" + _vers
-        for _d in _dsets:
-            depends_on(_d, type=("build", "run"), when=_vers)
+        for _dv in _dsets:
+            _d, _v = _dv.split("@")
+            variant(_d, default=True, when=_vers, description="Install dataset " + _d)
+            depends_on(_dv, type=("build", "run"), when=_vers)
```

Comment:
Kinda essential, but forgot to copy into the PR:
```suggestion
            depends_on(_dv, type=("build", "run"), when=_vers + " +" + _d)
```